frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.unit.spec.tsx

```diff
@@ -163,6 +163,75 @@ describe("DatabaseEditApp/Sidebar", () => {
     });
   });
 
+  describe("model actions control", () => {
+    it("is shown if database supports actions", () => {
+      setup();
+      expect(screen.getByLabelText(/Model actions/i)).toBeInTheDocument();
```

Comment:
I wonder if we could just use a simple string rather than a regex because string look simpler?

Reply:
The only reason to use regexps here is because of the `i` flag at the end, making the query case-insensitive, so in case we tweak the copy we don't need to always patch tests and let them focus on logic and semantics. This is something I subconsciously picked up from Kent C Dodd's demos of the testing library, but haven't put too much thought into this. Do you think case-sensitive queries and lack of slashes make more sense? I don't have a strong opinion on this

Reply:
Ahhh. That makes sense. I think we should keep using this then 👍